modules/nextflow/src/main/groovy/nextflow/k8s/model/PodSpecBuilder.groovy

```diff
@@ -573,7 +573,11 @@ class PodSpecBuilder {
         // -- host path volumes
         for( PodHostMount entry : hostMounts ) {
             final name = nextVolName()
-            mounts << [name: name, mountPath: entry.mountPath]
+            def nextMount = [name: name, mountPath: entry.mountPath]
+            if( entry.readOnly )
+                nextMount.readOnly = true
+
+            mounts << nextMount
```

Comment:
Likely `nextMount` is not needed, and the change can be applied directly to `mounts`

Reply:
Got it, you are right